soh/soh/Enhancements/cosmetics/CosmeticsEditor.cpp

```diff
@@ -36,8 +36,13 @@ const char* MarginCvarList[] {
 ImVec4 GetRandomValue(int MaximumPossible){
     ImVec4 NewColor;
     unsigned long range = 255 - 0;
+    #ifndef __SWITCH__
     std::random_device rd;
     std::mt19937 rng(rd());
+    #else
+    size_t seed = std::hash<std::string>{}(std::to_string(rand()));
```

Comment:
can we seed with time instead?

Reply:
I run srand with time at a different point in the code. I needed to srand once for every thing to randomize so I needed to do that part in a different place, otherwise everything became the same randomized color.

Reply:
See line 54. Unless I'm misunderstanding how the randomization works.

Reply:
do we call `GetRandomValue` from anywhere other than `GetRandomColorRGB`?

Reply:
We do now, when you click the Random button for an individual UI element it calls a different function, which I call GetRandomValue in there.

Reply:
Now that I think about it though, I'm seeing that I'm potentially not calling srand in that case. I may be able to refactor and reduce some duplication here.

Reply:
Nevermind on the reducing duplication, there's some poorly named functions in here that led me to believe I could do something different. Might need more refactoring here later but it's out of scope for now.

Reply:
sounds like a plan, would you mind making an issue for future cleanup?